src/main/java/uk/gov/ons/ctp/response/action/service/CaseNotificationService.java

```diff
@@ -82,38 +86,64 @@ public void acceptNotification(final CaseNotification notification) throws CTPEx
     actionCaseRepo.flush();
   }
 
-  private ActionCase createActionCase(CaseNotification notification) {
-    UUID actionPlanId = UUID.fromString(notification.getActionPlanId());
-    ActionPlan actionPlan = actionPlanRepo.findById(actionPlanId);
-    if (actionPlan == null) {
-      throw new IllegalStateException(
-          String.format(ACTION_PLAN_NOT_FOUND, actionPlanId.toString()));
+  private ActionCase createActionCase(CaseNotification notification) throws NullPointerException {
+    if (notification.getActionPlanId() != null) {
```

Comment:
Could we have a comment here to remind us to remove this code once we get rid of Action Plans?

Reply:
Sure 👍